index.js

```diff
@@ -1095,10 +1098,15 @@ Multi.prototype.exec = function (callback) {
                     args[args.length - 1](null, reply);
                 }
             }
+        } else if (err) {
+            err.code = 'EXECABORT';
+            err.errors = errors;
         }
 
         if (callback) {
-            callback(null, replies);
+            callback(err, replies);
```

Comment:
I'm uncertain if this should stay like that. Likely not. It might be nice to see what commands passed but this might be confusing if both is returned at the same time.